src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -21,10 +21,15 @@ public sealed class CSharpParseOptions : ParseOptions, IEquatable<CSharpParseOpt
         private ImmutableDictionary<string, string> _features;
 
         /// <summary>
-        /// Gets the language version.
+        /// Gets the effective language version.
         /// </summary>
         public LanguageVersion LanguageVersion { get; private set; }
 
+        /// <summary>
+        /// Gets the specified language version.
+        /// </summary>
+        public LanguageVersion SpecifiedLanguageVersion { get; private set; }
```

Comment:
>SpecifiedLanguageVersion [](start = 31, length = 24)

Could you explain what is the difference between values stored in LanguageVersion and SpecifiedLanguageVersion?